### PR TITLE
Fix the versions in the conda environment to ones that are currently known to work

### PR DESCRIPTION
**Pull request by cianwilson**
Only mac so far, will give linux a quick whirl now.

Removed duplicate dependency on sphinx, added in vtk and funcsigs to allow generic input tests to run.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -2,19 +2,21 @@ name: radev
...
-  - fftw
-  - mpich
+  - python=3.9
+  - make=4.3
+  - git=2.30
+  - matplotlib=3.4
+  - jupyter=1.0
+  - scipy=1.6
+  - sphinx=4.0
+  - sphinxcontrib-bibtex=2.2
```
Comment (gassmoeller) [inline environment.yml:12]: Did we resolve the issue with sphinxcontrib-bibtex version larger than 2.0 discussed earlier this week: https://rayleighworkspace.slack.com/archives/C022V39JPMG/p1621870838001100?

If not, should we fix the version to 1.0.0 like @feathern suggested in the slack thread? I somehow lost track of the issue and do not remember our solution.

**Review by gassmoeller — COMMENTED** [review]
Looks good, thanks for doing this! I have one question, otherwise this looks ready.

File: environment.yml (hunk 1 above)
Comment (cianwilson) [reply environment.yml:12]: I believe we were able to upgrade the version following #269 and no longer pin that package.

I've tested `make doc` on both mac and linux with these files and didn't spot anything going wrong (though I only looked at the pdf).

Comment (gassmoeller) [reply environment.yml:12]: Ah thanks, I must have missed #269 somehow. Ready to go then.

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**